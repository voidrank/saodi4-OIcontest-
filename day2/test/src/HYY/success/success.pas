program x;
  const 
    f1='success.in';
	f2='success.out';
  var
    n,j:longint;
	sum,x1:real;
  function fuck(i:longint):real;
    var 
	  total:real;
	begin 
	  total:=1;
	  for j:=1 to i do
	  total:=total*x1;
	  fuck:=x1/i;
	end;
  procedure fuck_off;
    begin 
	  for j:=1 to n do
	  sum:=sum+fuck(j);
	end;
begin 
  assign(input,f1);
  assign(output,f2);
  reset(input);
  rewrite(output);
  
  read(x1,n);
  sum:=0;
  fuck_off;
  write(sum:0:4);

  close(input);
  close(output);
end.