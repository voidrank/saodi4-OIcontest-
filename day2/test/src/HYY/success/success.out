 1.0E+000